src/resources/Namespaces/NamespaceDetails.js

```diff
@@ -89,7 +89,7 @@ export function NamespaceDetails(props) {
       customColumns={customColumns}
       headerActions={headerActions}
     >
-      <div className="panel-grid">
+      <div className="panel-grid bsl-margin-top--md">
```

Comment:
Please import spacing from UI5 and make it like this
```suggestion
          <div className="panel-grid" style={spacing.sapUiMediumMarginTop}>
```